### DISCUSSION TITLE
Chatterino User Survey

### DISCUSSION THREAD

**Discussion by gempir** [Chatterino/chatterino2#3162] [Ideas]
I was thinking that's really hard to tell what users use chatterino actually for or how they use chatterino.
Chatterino has no telemetrics and I don't think it should add them.

But to gain some helpful information about our users I propose a "Chatterino User Survey"

This can either be a time-limited survey, or just a permanent thing that can be used to collect feedback.

A short-ish survey that covers most important aspects. Examples for questions:

- What are the 3 biggest issues with Chatterino for you?
- Which Settings have you adjusted?
    -  Looks (Timestamps, Badges, Theme etc)
    -  Highlights
    -  Filters
    -  Commands
-  Features Wishes (Free Text?)
- Which Operating System do you use?
- ... ? 

I don't think this should be integrated into Chatterino, just spread around via word of mouth and maybe a bit of socials.

**Comment by Mm2PL** [comment]
I think it would be nice to have a form for this. Reducing activity of trolls might be hard without requiring logging in (be it Twitch or Github). We could just use Github issues but that's not very user-friendly.

I believe this might also be a nice way to check how many users of specific OSes we have.

**Reply by gempir** [reply]
I think I would offer myself to write a neat frontend on the site, but we'd need some sort of Backend to save data. I was also thinking Github and when a user wants to retract data he can simply click and his issue gets deleted or something. 

Github login is sadly gonna annoy a lot of people and just straight up exclude a lot of people. Twitch would be great but we can't store any data on twitch really.

**Reply by gempir** [reply]
Maybe just do a Google Form. I'm overthinking the technical part.

What's more important is the contents of the survey that should be the center of discussion

**Reply by fourtf** [reply]
Why write a frontend when there is things like google forms?
Also is there any actual knowledge on how to design a user survey?
Is it clear what the survey is trying to accomplish?
Sorry but jumping to "how to implement it" immediately is right approach.

**Reply by gempir** [reply]
Yes. Exactly. Ignore the technical part. We can imagine it's just a Google Form for now.

I have limited experience on the matter, that's why I started the discussion.

**Comment by apa420** [comment]
What do you want to do with the information you get out of this?
From my point of view most of these "questions" are quite useless.
I would instead maybe try and get people to :+1: the issues/ideas or bugs that they would like to get fixed inside of github.
This would make it clear for people implementing features or fixing bugs what people want more than a survey would imo.

>Features Wishes (Free Text?)

Isn't this literally github discussions?

Also I don't really see any point in asking what operating system people use.
I don't see it changing development in any shape really.
Maybe system-specific issues? but there aren't that many of them other than high dpi ones.
Might be smarter to ask if they use high dpi screens or something.

The people who don't use github right now instead go and ask in example #pajlada about an bug or feature they want added and someone with a github account creates an issue/discussion. Maybe this could be improved instead?

tldr;
I would find a survey useless and would rather maybe promote people using github to vote on stuff they want fixed/added.
